cluster/nginx.go

```diff
@@ -254,7 +299,7 @@ func UpdateNginxConfiguration(ctx *Context) chan error {
 					location / {
 						 proxy_set_header Upgrade $http_upgrade;
 						 proxy_set_header Connection "upgrade";
-						 client_max_body_size 50M;
+						 client_max_body_size 0;
```

Comment:
0 means no limit in the body size?  if yes I think we should declare a maximum limit, even if this is big , to avoid potential DOS

Reply:
That would be 5TiB single stream.